### PR TITLE
Common-utils Feature: Pin to major version 2

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Ref: https://github.com/devcontainers/features/pull/326

### CODE REVIEW

**Review by eitsupi — COMMENTED** [review]
I believe the `uid` and `gid` option names have changed, so must be update here. (same #308)
https://github.com/devcontainers/features/blob/a9a7a25d60585ce2145374ec6825cbdc7f16cb90/src/common-utils/devcontainer-feature.json#L40-L57

I am also wondering if it is possible to change the `meta.env`'s path.
https://github.com/devcontainers/features/blob/a9a7a25d60585ce2145374ec6825cbdc7f16cb90/src/common-utils/bin/devcontainer-info#L7

**Comment by samruddhikhandale** [comment]
> I believe the uid and gid option names have changed, so must be update here. (same https://github.com/devcontainers/images/pull/308)
https://github.com/devcontainers/features/blob/a9a7a25d60585ce2145374ec6825cbdc7f16cb90/src/common-utils/devcontainer-feature.json#L40-L57

Yep, thanks for catching. Updated

**Comment by samruddhikhandale** [comment]
> I am also wondering if it is possible to change the meta.env's path.
https://github.com/devcontainers/features/blob/a9a7a25d60585ce2145374ec6825cbdc7f16cb90/src/common-utils/bin/devcontainer-info#L7

I think we could update that, it would mostly need an update to the vscdc tool [here](https://github.com/devcontainers/images/blob/main/build/src/prep.js#L56). @Chuxel any concerns?

**Comment by Chuxel** [comment]
> > I am also wondering if it is possible to change the meta.env's path.
> > https://github.com/devcontainers/features/blob/a9a7a25d60585ce2145374ec6825cbdc7f16cb90/src/common-utils/bin/devcontainer-info#L7
> 
> I think we could update that, it would mostly need an update to the vscdc tool [here](https://github.com/devcontainers/images/blob/main/build/src/prep.js#L56). @Chuxel any concerns?

Yep. Looks like @eitsupi did a PR already to update the common-utils `devcontainer-info` command to check the `dev-containers` folder in addition to `vscode-dev-containers`, so updating the image generation should be all that is required!

**Comment by samruddhikhandale** [comment]
> > > I am also wondering if it is possible to change the meta.env's path.
> > > https://github.com/devcontainers/features/blob/a9a7a25d60585ce2145374ec6825cbdc7f16cb90/src/common-utils/bin/devcontainer-info#L7
> > 
> > 
> > I think we could update that, it would mostly need an update to the vscdc tool [here](https://github.com/devcontainers/images/blob/main/build/src/prep.js#L56). @Chuxel any concerns?
> 
> Yep. Looks like @eitsupi did a PR already to update the common-utils `devcontainer-info` command to check the `dev-containers` folder in addition to `vscode-dev-containers`, so updating the image generation should be all that is required!

Great! I'll create another PR once this one's merged, thanks!

**Review by edgonmsft — DISMISSED** [review]

File: src/base-debian/.devcontainer/devcontainer.json [hunk 1]
```diff
@@ -4,11 +4,11 @@
 		"context": "."
 	},
 	"features": {
-		"ghcr.io/devcontainers/features/common-utils:1": {
+		"ghcr.io/devcontainers/features/common-utils:2": {
```
Comment (samruddhikhandale) [inline src/base-debian/.devcontainer/devcontainer.json:7]: Blocked by https://github.com/devcontainers/features/pull/388

**Pull request merged**